build.properties.default

```diff
@@ -286,14 +286,14 @@ saaj-api.home=${base.path}/saaj-api-${saaj-api.version}
 saaj-api.jar=${saaj-api.home}/saaj-api-${saaj-api.version}.jar
 saaj-api.loc=${base-maven.loc}/javax/xml/soap/saaj-api/${saaj-api.version}/saaj-api-${saaj-api.version}.jar
 
-# ----- bnd & bndlib, version 4.0.0 or later  -----
+# ----- bnd, version 5.1.1 or later  -----
```

Comment:
Is this really an increase in the minimum version? Or is 5.1.0 (with the fix for the Java 15 bug) sufficient?

Reply:
There is that yes, but also in 5.1.1 specifically there is a bug fix for correctly generating the Manifest Name section which I'd like to use for the Spec jars.

Reply:
Fair enough. Just checking.